oauth2_http/java/com/google/auth/oauth2/ExternalAccountCredentials.java

```diff
@@ -280,8 +294,6 @@ protected ExternalAccountCredentials(ExternalAccountCredentials.Builder builder)
         builder.metricsHandler == null
             ? new ExternalAccountMetricsHandler(this)
             : builder.metricsHandler;
-
-    this.impersonatedCredentials = buildImpersonatedCredentials();
```

Comment:
Sure, probably makes sense to just to keep it the same. I didn't make that change initially since the issue we have with the impersonated credentials is only possible in the builder constructor